src/omv/imlib/framebuffer.c

```diff
@@ -326,24 +309,21 @@ void framebuffer_flush_buffers(bool fifo_flush) {
 }
 
 int framebuffer_set_buffers(int32_t n_buffers) {
-    uint32_t total_size = framebuffer_raw_buffer_size();
-    uint32_t size = total_size / n_buffers;
+    uint32_t avail_size = FB_ALIGN_SIZE_ROUND_DOWN(framebuffer_max_buffer_size());
```

Comment:
```suggestion
    uint32_t avail_size = FB_ALIGN_SIZE_ROUND_DOWN(framebuffer_max_buffer_size() - sizeof(vbuffer_t));
```

One frame buffer still has a vbuffer header that needs to be removed.